### PR TITLE
authors: add Yufen Huang to authors

### PR DESCRIPTION
**Pull request by Elliebababa**
Release note: None

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -338,6 +338,7 @@ Yang Yuting <william.yangyt@gmail.com>
 yangliang9004 <yangliang9004@gmail.com>
 Yevgeniy Miretskiy <yevgeniy@cockroachlabs.com> <yevgeniy@gmail.com>
 Yosi Attias <yosy101@gmail.com>
+Yufen Huang <ellie24.huang@gmail.com>
```
Comment (otan) [inline AUTHORS:341]: add your cockroach email too!

**Review by otan — APPROVED** [review]

**Comment by Elliebababa** [comment]
bors r=otan

**Comment by Elliebababa** [comment]
bors r=otan

**Comment by Elliebababa** [comment]
bors r=otan

**Pull request merged**